init/spark/spark-defaults.conf

```diff
@@ -27,5 +27,5 @@ spark.sql.catalog.catalog_rest.type rest
 spark.sql.catalog.catalog_rest.uri http://gravitino:9001/iceberg/
 spark.locality.wait.node 0
 spark.sql.warehouse.dir hdfs://hive:9000/user/hive/warehouse
-spark.sql.hive.metastore.jars path
-spark.sql.hive.metastore.jars.path file:///opt/spark/jars/*
+spark.sql.catalog.catalog_hive.spark.sql.hive.metastore.jars path
```

Comment:
Is this Kyuubi config option?

Reply:
It's a spark option to bypass the problems that spark-sql client querying multi catalogs, see https://github.com/apache/kyuubi/issues/6362